tests/e2e_test.go

```diff
@@ -344,8 +350,32 @@ func TestIntoto(t *testing.T) {
 	// If we do it twice, it should already exist
 	out := runCli(t, "upload", "--artifact", attestationPath, "--type", "intoto", "--public-key", pubKeyPath)
 	outputContains(t, out, "Created entry at")
+	uuid := getUUIDFromUploadOutput(t, out)
+
+	// The atteestation should be stored at /var/run/attestations/$uuid
+	cmd := exec.Command("docker", "run", "-v", "/var/run/attestations:/var/run/attestations", "alpine", "cat", "/var/run/attestations/"+uuid)
```

Comment:
could this be simplified to just read directly from `/var/run/attestations/` on the host?

Reply:
Not always, especially in CI you're typically running in a container too with the docker socket mounted in.

It's the same roughly for Mac or anywhere that you have docker in a VM.